core/src/tpu.rs

```diff
@@ -228,12 +246,29 @@ impl Tpu {
         .unwrap();
 
         let (forward_stage_sender, forward_stage_receiver) = bounded(1024);
-        let sigverify_stage = {
+        let sig_verifier = if let Some(vortexor_receivers) = vortexor_receivers {
```

Comment:
I think we should optionally spawn vortexor receiver, but we should still spawn the local sigverify stage.

AFAICT the quic and fetch stages still exist, and even if the ports are not advertised on gossip it's possible to still send to them.

This also makes upgrades tied together. I can't restart my vortexor instance because my validator is relying on it.
We should let the operator switch their advertised tpu port(s) at runtime, so they can switch back to local mode to upgrade vortexor without screwing up the validator.
Or switch to another vortexor instance.

Reply:
Great point. We plan to support the dynamic management of subscriptions via Admin RPC on the validator. See https://github.com/anza-xyz/agave/blob/master/vortexor/Readme.md

Reply:
Good point! Will address

Reply:
Disable TPU streamers when vortexor receiver is configured.

Reply:
Sorry for the delay, but this seems the wrong direction to me.
If the vortexor goes down, the operator is forced to restart their node to go back to normal TPU. This seems like a necessary feature to me, not something that we should do as follow-up.

I'm happy to hear other's opinions, as maybe I'm being overly cautious about this.

Reply:
This feature is only enabled when tpu-vortexor-receiver-address enabled. Auto fullback feature will be delivered in follow-on PRs.